hardhat.config.ts

```diff
@@ -62,14 +75,21 @@ const config: HardhatUserConfig = {
         "node_modules/@keep-network/tbtc/artifacts",
         "./external/ropsten",
       ],
+      mainnet: ["./external/mainnet"],
```

Comment:
This solution assumes the artifacts from `keep-core` and `tbtc` that we need addresses for are committed to the repository. I decided to go with it to reduce the possibility of using wrong addresses if we use npm packages.

An alternative way would require executing `yarn add @keep-network/keep-core@1.7.0 @keep-network/tbtc@1.1.0` to install packages from specific mainnet versions and update this line to:
```ts
      mainnet: [
        "node_modules/@keep-network/keep-core/artifacts",
        "node_modules/@keep-network/tbtc/artifacts",
        "./external/mainnet"
      ],
```

Reply:
I think the approach you used is better. We need them only for the deployment. Does it mean we can remove them from `package.json`? 
```
    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-pre",
    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
```

Reply:
Ah, no, we can't because they are used for `development` network earlier, right? So ones in `package.json` should always stay dev-only. If that's correct, can we drop a comment in `package.json` explaining these dependencies - why they are needed, where they are referenced, and what versions (dev-only) are expected?

Reply:
They are used for development and will be used for ropsten. Basically, we need them in `package.json` for any network that artifacts will be changing dynamically. For `ropsten` we will be updating the package version to a specific one deployed on ropsten.

Unfortunately, we cannot add comments in a JSON file :( But this approach is a general approach that we have in all our repositories.